x-pack/plugins/observability/public/components/app/section/uptime/index.tsx

```diff
@@ -75,7 +75,7 @@ export function UptimeSection({ bucketSize }: Props) {
     ]
   );
 
-  if (!hasDataMap.synthetics?.hasData) {
+  if (!hasDataMap.uptime?.hasData) {
```

Comment:
Not necessarily related to this line, but will we need to add a Synthetics section to the observability overview page before GA?